parsl/serialize/concretes.py

```diff
@@ -0,0 +1,61 @@
+import dill
+import pickle
+import logging
+
+logger = logging.getLogger(__name__)
+from parsl.serialize.base import SerializerBase
+
+
+class PickleSerializer(SerializerBase):
+    """ Pickle serialization covers most python objects, with some notable exceptions:
+
+    * functions defined in a interpretor/notebook
+    * classes defined in local context and not importable using a fully qualified name
+    * clojures, generators and coroutines
+    * [sometimes] issues with wrapped/decorated functions
+    """
+
+    _identifier = b'01\n'
+    _for_code = True
+    _for_data = True
+
+    def __init__(self):
```

Comment:
this init could go away because it does nothing other than the default behaviour, right?